src/dbus/server/introspect.xml

```diff
@@ -337,6 +356,26 @@
     <property name="ExternalRoutes" type="((ayy)qybb)" access="read">
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
     </property>
+
+    <signal name="CommissionerState">
+      <arg type="y" name="state"/>
+    </signal>
+
+    <signal name="JoinerEvent">
```

Comment:
Make joiners a property of the commissioner object?

Reply:
Joiner event should not be a property. One joiner will issue 4 to 5 different events.